### PR TITLE
Grafana agent kubelet metrics

### PR DESCRIPTION
**Pull request by jimgus**
Make it possible to ship Kubelet metrics for tenant namespaces to Grafana cloud.

### CODE REVIEW

**Review by NissesSenap — APPROVED** [review]

**Comment by phillebaba** [comment]
Should we have some protection against for example accidentally exposing kube-system?

**Comment by jimgus** [comment]
> Should we have some protection against for example accidentally exposing kube-system?

You a re thinking about a check that tenant namespaces and extra namespaces don't contain "kube-system"?

**Comment by phillebaba** [comment]
Yes, might be useless but just an idea.

**Comment by NissesSenap** [comment]
It's a fair point but I would argue that it's out of scope for this PR and should be in another one. Feel free to create an issue/PR about it @phillebaba

**Review by phillebaba — APPROVED** [review]
LGTM

**Review by NissesSenap — APPROVED** [review]

File: modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/monitors.yaml [hunk 1]
```diff
@@ -19,3 +19,94 @@ spec:
...
+        - action: keep
+          regex: {{ .Values.kubeletMetricsNamespaces }}
+          sourceLabels:
+            - namespace
+        - action: keep
+          regex: container_cpu_usage_seconds_total|container_network_transmit_bytes_total|container_network_receive_bytes_total|container_memory_swap|container_network_receive_packets_total|container_cpu_cfs_periods_total|container_cpu_cfs_throttled_periods_total|node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate|container_memory_working_set_bytes|container_memory_rss|container_network_receive_packets_dropped_total|container_start_time_seconds|container_network_transmit_packets_dropped_total|container_memory_cache|container_network_transmit_packets_total
+          sourceLabels:
+            - __name__
+        - action: labeldrop
+          regex: endpoint|id|job|metrics_path|service|name|instance
```
Comment (NissesSenap) [inline modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/monitors.yaml:45]: Don't know how the final output looks. 
Can't remember what service says but can it be useful?

Comment (jimgus) [reply modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/monitors.yaml:45]: Not really, it is always set to "kubelet".

**Pull request merged**